src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -249,15 +249,14 @@ struct LottieGenericProperty : LottieProperty
         return frame->interpolate(frame + 1, frameNo);
     }
 
-    T& operator=(const T& other)
+    void shallowCopy (const LottieGenericProperty<T>& other)
     {
         //shallow copy, used for slot overriding
         delete(frames);
```

Comment:
You should not delete the actual property data for backup(reverting) ??

Reply:
Yes. It's the revising operator commit.
The deletion in shallow copy case has been removed in next commit.